00new_packages.autobuild

```diff
@@ -1,4 +1,7 @@
 # IMPORTANT: new packages must be added at the top of this file
 # They must NEVER be added to other files
 
-
+cmake_package 'gui/rock_webapp'
+ruby_package 'tools/rest_api' do |pkg|
+    Autoproj.env_add_path 'ROCK_WEBAPP', File.join(pkg.prefix, 'share', 'rest_api')
```

Comment:
Sorry, but this is not what it is anymore. Currently ROCK_WEBAPP (still without the path, but have no problem changing it) points to install/share/rest_api. Rock-webapp will look for any *.rb files in install/share/rest_api/plugin. Inside these plugins, you can modify the rack mappings, and add search paths for sprockets (which is used to serve and preprocess js and css). The way I currently did it that the mapping for e.g. for the webapp would point to the public directory of the webapp source package (or whichever other package).